cpp/examples/string_transforms/branching.cpp

```diff
@@ -139,7 +139,9 @@ __device__ void format_phone(void* scratch,
 
   auto const num_rows = table.num_rows();
   rmm::device_uvector<char> scratch(
-    MAX_ENTRY_LENGTH * num_rows, stream, mr);  // allocate scratch space for the outputs
+    static_cast<std::size_t>(MAX_ENTRY_LENGTH) * static_cast<std::size_t>(num_rows),
```

Comment:
```suggestion
    MAX_ENTRY_LENGTH * static_cast<std::size_t>(num_rows),
```
I think you may only need one of these to be cast.

Reply:
I'm pretty sure it will convert the `int` to `unsigned-long (size_t)` without complaint. 
Regardless, casting both is always safe.